Screen_OLED_Test/OLEDScreen.cpp

```diff
@@ -68,32 +67,50 @@ int elapsedTimeUpdateMillis = 0;
 unsigned long previousTimeUpdateMillis = 0;
 float percentageOfSecondElapsed = 0;
 
-bool directionIO;
+bool directionForwBack;
+bool directionLeftRight;
+
+/* ================= Debugging variables ================ */
+int testingDelay = 0;
+int testingState = 0;
+int testHighlightModeState = 0;
+int testBatteryBarState = 0;
+int testArrowHighlight = 0;
+
+unsigned long testPreviousMillis = 0;
+const long testInterval = 100;  // Interval in milliseconds (adjust as needed)
 
 int readInt(int address) {
   return word(EEPROM.read(address), EEPROM.read(address + 1));
 }
 
-void setDirection(bool newDirection) {
-  directionIO = newDirection;
+void setForwBackDirection(bool newForwBackDirection) {
+  directionForwBack = newForwBackDirection;
+}
+
+void setLeftRightDirection(bool newLeftRightDirection){
+  directionLeftRight = newLeftRightDirection;
 }
 
-void initIOModule(int _modeSwitchButton, bool initDirection) {
+void initIOModule(int _modeSwitchButton, bool initForwBackDirection, bool initLeftRightDirection, bool debug) {
   modeSwitchButton = _modeSwitchButton;
-  setDirection(initDirection);
+  setForwBackDirection(initForwBackDirection);
+  setLeftRightDirection(initLeftRightDirection);
   u8g2.setColorIndex(1);
   u8g2.setFont(u8g2_font_freedoomr10_tu);
   u8g2.begin(); // begin u8g2 library
   u8g2.setBitmapMode(1);
   pinMode(modeSwitchButton, INPUT_PULLUP);
   // pinMode(A0, INPUT); Potentiometer, maybe for debug
-  /* pinMode(button2, INPUT_PULLUP); // Other buttons for Debug
-  pinMode(buttonUp, INPUT_PULLUP);
+  /*pinMode(buttonUp, INPUT_PULLUP);
```

Comment:
Don't leave commented code. If you need these for debugging purposes then create an if condition with the bool debug. Depending on that we either initalise them or not